src/audio/Kconfig

```diff
@@ -51,6 +51,14 @@ config COMP_UP_DOWN_MIXER
          Downmixing for mono output:
          4.0, Quatro, 3.1, 2 -> 1
 
+config COMP_BLOB
+	bool "Large IPC data as compound message blobs"
+	default y
+	help
+	 Select to support sending large data blobs to firmware as
+	 multiple IPC messages. Not all components or modules need
+	 this. If unsure, say yes.
+
```

Comment:
Shouldn't the current users be made dependent on this? If you choose a component that requires the blob failure, you'll build will now fail unless this is selected.

Reply:
yep, I will add this. I think it's mainly stuff we use in the newer products.